app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
```

Comment:
what does this do?

Reply:
To tell that this class will share a database connection with Active Record